src/helpers/getFixtureInfo.js

```diff
@@ -1,15 +1,20 @@
+import path from 'path';
+
 export default function getFixtureInfo(fixtureInput) {
   if (typeof fixtureInput === 'string') {
     return {
       filePath: fixtureInput,
       encoding: '',
       mimeType: '',
+      fileName: path.basename(fixtureInput),
     };
   }
 
   return {
     filePath: fixtureInput.filePath,
     encoding: fixtureInput.encoding || '',
     mimeType: fixtureInput.mimeType || '',
+    fileName: fixtureInput.fileName || path.basename(fixtureInput),
+    fileContent: fixtureInput.fileContent,
```

Comment:
I think this change should also be reflected in `index.d.ts`

Reply:
https://github.com/abramenal/cypress-file-upload/blob/master/types/index.d.ts#L4

Reply:
I have updated the type definitions, and also added a validation rule in order to avoid the scenario where a person would pass `fileContent` and not provide either `fileName` or `filePath`

Reply:
Great!
Last thing I forgot to mention – please describe these new properties in `README.md` so that other people understand how to use it. I think it's best to include 1-2 examples of using different styles (I mean `fileName` vs `filePath` vs `fileContent + fileName`

Reply:
I am not sure if you did this, and I blame myself for not configuring CI properly, but just for your safety you can use recipes to actually test whether that works.
We can update recipes later on, right now the primary focus is fixing the issue and unblocking users

Reply:
I think you should now be able to merge this

Reply:
Updated the `README.md` let me know if is ok like that or you want it to be more verbose. I'm a little bit pressed by time at the moment but can make it more clear later today.

I did test the code but not using recipes, since I did not notice that folder until now. What I did was create the bundle and put it in the `node_modules/cypress-file-upload` of a project were I'm using the plugin. 

I can use the recipes later today to confirm everything works as expected but I'm quite sure it will.

Reply:
I think it's perfect – let's merge it to master, and I can run the recipes and let you know if there's anything missed here.
Thank you so much for the contribution!